### PR TITLE
Update maven config for server package

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: util/pom.xml [hunk 1]
```diff
@@ -73,18 +73,6 @@
 
     <build>
         <plugins>
-            <plugin>
```
Comment (shifucun) [inline util/pom.xml:76]: removed these since there is no change from parent

Comment (pradh) [reply util/pom.xml:76]: I remember I had some issue with things not working unless I included it, and so I added it...

**Review by pradh — APPROVED** [review]
Just to sanity check, if you do mvn package, then run lint on an example MCF/TMCF/CSV does it run okay?

**Comment by pradh** [comment]
Nice!

**Comment by shifucun** [comment]
> Just to sanity check, if you do mvn package, then run lint on an example MCF/TMCF/CSV does it run okay?

Tested locally, and it worked fine.

**Comment by shifucun** [comment]
> Nice!

Thanks for review!

File: util/pom.xml (hunk 1 above)
Comment (pradh) [reply util/pom.xml:76]: Can we add this back?

With this change it seems the following don't seem to be included when I run `mvn compile`
```
[INFO] --- protoc-jar-maven-plugin:3.11.4:run (default) @ datacommons-import-util ---

[INFO] --- fmt-maven-plugin:2.11:format (default) @ datacommons-import-util ---
```

**Pull request merged**